docs/administrator-documentation/moderne-dx/how-to-guides/configure-dx-organizations.md

```diff
@@ -209,3 +165,12 @@ mod git clone moderne <path> <organization-id>
 ```
 
 Once these are both done, your developers can now start running recipes and committing the results!
+
+## Additional customization
+
+You may want to create a dedicated organizations service if you want to:
+
+* Limit access to the organizations you've defined above so that some users only have access to some repositories 
+* Customize commit messages by repository (such as adding a JIRA ticket per repository)
```

Comment:
DX doesn't have the ability to customize commit messages by repository, so I would suggest we simplify to this:

```suggestion
You may want to create a dedicated organizations service if you want to limit access to the organizations you've defined above so that some users only have access to some repositories.
```